examples/word_finder.md

```diff
@@ -0,0 +1,157 @@
+# Building a Word Finder App with Deno
+
+## Getting Started
+
+In this tutorial we'll create a simple Word Finder web application using Deno.
+No prior knowledge of Deno is required.
+
+## Introduction
+
+Our Word Finder application will take a pattern string provided by the user and
+return all words in the English dictionary that match the pattern. The pattern
+can include alphabetical characters as well as `_` and `?`. The `?` can stand
+for any letter that isn't present in the pattern. `_` can stand for any letter.
+
+For example, the pattern `c?t` matches "cat" and "cut". The pattern `go?d`
+matches the words "goad" and "gold" (but not "good").
+
+![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/fe855a02-b26a-4d80-b601-65bd31cbc099/Untitled.png)
+
+## Building the View
+
+We can use EJS to create a simple UI for our Word Finder app. EJS is a
```

Comment:
I'm really concerned about this example, as it doesn't appear to work out of the box under Deno.  I don't think we should be promoting EJS templating in Deno either.  EJS !== JSX and so these are failing checking because they aren't supported natively.  The code here doesn't match the repository code either.

Reply:
Ah sorry, Akila actually updated the example in Notion with vanilla js but I had copied over the old version. That should match the repository code.